nipype/interfaces/base.py

```diff
@@ -1616,11 +1638,9 @@ def _format_arg(self, name, trait_spec, value):
             # Depending on whether we stick with traitlets, and whether or
             # not we beef up traitlets.List, we may want to put some
             # type-checking code here as well
-            sep = trait_spec.sep
-            if sep is None:
-                sep = ' '
-            if argstr.endswith('...'):
+            sep = trait_spec.sep or ' '
```

Comment:
This will replace `''` with `' '`. I'm not sure that's what people will expect. Actually equivalent:

```Python
sep = trait_spec.sep if trait_spec.sep is not None else ' '
```